lib/cmds/survey_cmds/export-responses.js

```diff
@@ -0,0 +1,112 @@
+'use strict';
+
+const querystring = require('querystring');
+const { get } = require('../../api');
+const plainPrint = require('../../plainPrint');
+
+exports.command = 'export-responses <projectId> <surveyId>';
+exports.desc = 'Export all responses in .csv format';
+exports.builder = yargs => {
+  yargs.positional('projectId', {
+    describe: 'The ID of the project containing the survey.',
+    type: 'string'
+  }).positional('surveyId', {
+    describe: 'The ID of the survey to export responses from.',
+    type: 'string'
+  });
+};
+
+function buildQuestionList (surveyItem, results) {
+  if (surveyItem.linkId && surveyItem.text && surveyItem.type && surveyItem.type !== 'group' && surveyItem.type !== 'display') {
+    results.push({
+      text: surveyItem.text.replace(/\n/g, ' ').replace(/"/g, '""'),
+      linkId: surveyItem.linkId
+    });
+  }
+  if (surveyItem.item) {
+    (surveyItem.item || []).forEach(_ => buildQuestionList(_, results));
+  }
+}
+
+function generateLinkAnswerHash (responseItem, results) {
```

Comment:
Minor, but the name of this function confused me at first. I thought it was computing a hash value or something, but it's really building up an object/map. Probably better to name it `generateLinkAnswerMap`.